exercises/practice/pov/.meta/example.ex

```diff
@@ -0,0 +1,133 @@
+defmodule Pov do
```

Comment:
Since this exercise is basically `Zipper` + more functions, should see if we are being congruent with our approaches between the two exercises

Reply:
Coincidentally, I just reached the `Zipper` exercise in my Elixir track. I'll go solve it real quick before looking into the repo ;)

Reply:
OK, I've solved it now. The approach in `Zipper` is quite different from other exercises that I've seen so far since it provides a complete module for binary trees. It's not bad, should I model this one after `Zipper`?

Reply:
I think we can start with your current implementation like it is now, and then later, _if_ people complain that the exercise is too difficult, we could add hints in the form of a module with a data structure that we suggest using.

Reply:
Agreed, challenging exercises are not as much of an issue for practice exercises